File: Modules/LingT.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SteamKit2;

namespace STEAMNERD.Modules
{
    class LingT : Module
    {
        private readonly Random _rand;

        public LingT(SteamNerd steamNerd) : base(steamNerd)
        {
            _rand = new Random();
        }

        public override bool Match(SteamFriends.ChatMsgCallback callback)
        {
            var message = callback.Message.ToLower();
            return message.Contains("ling") && message.Contains("t");
        }

        public override void OnChatMsg(SteamFriends.ChatMsgCallback callback)
        {
            var stupidMessages = new[] {"selamat pagi", "ling t", "selamat malam"};
            SteamNerd.SendMessage(stupidMessages[_rand.Next(stupidMessages.Length)], callback.ChatRoomID, true);
        }
    }
}

File: Modules/LingT.py
import random

Module.Name = "Greetings"
Module.Description = "A friendly greeting."
Module.Global = True

var.greetings = ('selamat pagi', 'ling t', 'selamat malam')

def OnChatMessage(callback, args):
	message = callback.Message.lower()
	
	if message.find('ling t') != -1:
		SteamNerd.SendMessage(random.choice(var.greetings), callback.ChatRoomID)
